.dockerignore

```diff
@@ -0,0 +1,2 @@
+*/node_modules
```

Comment:
I think it's okay to remove those docker files

.dockerignore
Dockerfile
docker-compose.yml